tsconfig.json

```diff
@@ -48,7 +48,7 @@
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     // "typeRoots": [],                       /* List of folders to include type definitions from. */
     "types": [
-      "jest",
+      "vitest/globals",
```

Comment:
For some reason, vscode doesn't seem to pick these types up, when navigating files in the `test` folder. Can you add this line to the `fixture.ts` file?

```ts
import 'vitest/globals';
```

Reply:
This one is interesting - somehow it wasn't appearing in my environment due to an 'unknown to me' magical node_modules package cache living in my home directory. It was just resolving to the @types/jest that lived there.

Adding this resolves the vscode linting ... but breaks all of the tests which call `expectFunction()` from `fixture.ts`.

```term
 FAIL  packages/langium/test/lsp/completion-provider.test.ts [ packages/langium/test/lsp/completion-provider.test.ts ]
 FAIL  packages/langium/test/lsp/document-symbol.test.ts [ packages/langium/test/lsp/document-symbol.test.ts ]
 FAIL  packages/langium/test/lsp/folding-range.test.ts [ packages/langium/test/lsp/folding-range.test.ts ]
 FAIL  packages/langium/test/lsp/goto-definition.test.ts [ packages/langium/test/lsp/goto-definition.test.ts ]
 FAIL  packages/langium/test/lsp/hover.test.ts [ packages/langium/test/lsp/hover.test.ts ]
Error: No known conditions for "./globals" entry in "vitest" package
```

So, I'm at a bit of a loss - I went ahead and committed the code with the import masked incase someone had an idea. I'll keep poking at it to see if I can figure it out.